File: ANDROID PROGRAMMING WITH JAVA/PROJECTS/VehicleClaimApp_UserStory12/app/src/main/java/com/example/vehicleclaimapp_userstory12/service/ApiService.java
package com.example.vehicleclaimapp_userstory12.service;

import com.example.vehicleclaimapp_userstory12.model.CompanyUpdate;
import com.example.vehicleclaimapp_userstory12.model.Policy;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {
    @GET("company")
    Call<List<CompanyUpdate>> getCompanyUpdates();

    @GET("policy")
    Call<List<Policy>> getPolicyDeatils();
}

File: ANDROID WITH KOTLIN/PROJECTS/UserApp_RemoteAPI/app/src/main/java/com/example/userapp_remoteapi/api/ApiService.kt
package com.example.userapp_remoteapi.api

import com.example.userapp_remoteapi.model.User
import retrofit2.Call
import retrofit2.http.GET

interface ApiService {
    @GET("users")
    fun getUsers(): Call<List<User>>
    /*@GET("todos")
    fun getUsers(): Call<List<Todo>>*/
}
